### DISCUSSION TITLE
Internal module support similar to Go

### DISCUSSION THREAD

**Discussion by ghost** [vlang/v#11827] [Ideas]
Also, maybe be better not give this opportunity, because we have **one way** ideology.

**Comment by larpon** [comment]
Can you please specify what "folder will special opportunity" technically does in Go?

**Comment by ghost** [comment]
@Larpon [godock](https://golang.org/doc/go1.4#internalpackages)

**Comment by larpon** [comment]
Thanks for specifying 👍

**Comment by larpon** [comment]
As I understand it there's currently no plans for this feature.
